File: all_programs/C_Programs/stop-wait.py
def stop_wait():
    frame_size = int(input("Enter frame size : "))
    sent = 0
    while(sent<frame_size):
        print("Frame-" + str(sent) + " has been Transmitted.")
        ack = int(input("Enter Acknowledgement received : "))
        if(ack == sent):
            print("Ackowledgement received for Frame-" + str(sent))
            sent += 1
        else:
            print("Ackowledgement not received for Frame-" + str(sent))
    return

if __name__ == '__main__':
    stop_wait()

File: all_programs/C_Programs/stop_wait.c

//Sambit 

#include<stdio.h>
int main(){
    int framesize, sent = 0, ack, i;
    printf("Enter the frame size: ");
    scanf("%d", &framesize);
    while(sent < framesize) {
        printf("Frame %d sent\n", sent);
        printf("Waiting for acknowledgement...\n");
        scanf("%d", &ack);
        if(ack == sent) {
            printf("Acknowledgement received for frame %d\n", sent);
            sent++;
        }
        else {
            printf("Acknowledgement not received for frame %d\n", sent);
        }
    }
}